clang/test/Driver/sycl-offload-nvptx.cpp

```diff
@@ -52,5 +53,130 @@
+// RUN: -fsycl-targets=nvptx64-nvidia-cuda-sycldevice \
+// RUN: -Xsycl-target-backend "--cuda-gpu-arch=sm_35" -Xsycl-target-backend "--cuda-gpu-arch=sm_70" %s 2>&1 \
+// RUN: | FileCheck -check-prefix=CHK-PHASES-SM35-70 %s
+// CHK-PHASES-SM35-70: 0: input, "[[INPUT:.*]]", c++, (host-sycl)
+// CHK-PHASES-SM35-70: 1: preprocessor, {0}, c++-cpp-output, (host-sycl)
+// CHK-PHASES-SM35-70: 2: input, "[[INPUT]]", c++, (device-sycl, sm_70)
+// CHK-PHASES-SM35-70: 3: preprocessor, {2}, c++-cpp-output, (device-sycl, sm_70)
+// CHK-PHASES-SM35-70: 4: compiler, {3}, sycl-header, (device-sycl, sm_70)
+// CHK-PHASES-SM35-70: 5: offload, "host-sycl (x86_64-unknown-linux-gnu)" {1}, "device-sycl (nvptx64-nvidia-cuda-sycldevice:sm_70)" {4}, c++-cpp-output
+// CHK-PHASES-SM35-70: 6: compiler, {5}, ir, (host-sycl)
+// CHK-PHASES-SM35-70: 7: backend, {6}, assembler, (host-sycl)
+// CHK-PHASES-SM35-70: 8: assembler, {7}, object, (host-sycl)
+// CHK-PHASES-SM35-70: 9: linker, {8}, image, (host-sycl)
+// CHK-PHASES-SM35-70: 10: input, "[[INPUT]]", c++, (device-sycl, sm_35)
+// CHK-PHASES-SM35-70: 11: preprocessor, {10}, c++-cpp-output, (device-sycl, sm_35)
+// CHK-PHASES-SM35-70: 12: compiler, {11}, ir, (device-sycl, sm_35)
+// CHK-PHASES-SM35-70: 13: linker, {12}, ir, (device-sycl, sm_35)
+// CHK-PHASES-SM35-70: 14: sycl-post-link, {13}, ir, (device-sycl, sm_35)
+// CHK-PHASES-SM35-70: 15: backend, {14}, assembler, (device-sycl, sm_35)
+// CHK-PHASES-SM35-70: 16: assembler, {15}, object, (device-sycl, sm_35)
+// CHK-PHASES-SM35-70: 17: offload, "device-sycl (nvptx64-nvidia-cuda-sycldevice:sm_35)" {16}, object
+// CHK-PHASES-SM35-70: 18: offload, "device-sycl (nvptx64-nvidia-cuda-sycldevice:sm_35)" {15}, assembler
+// CHK-PHASES-SM35-70: 19: compiler, {3}, ir, (device-sycl, sm_70)
+// CHK-PHASES-SM35-70: 20: linker, {19}, ir, (device-sycl, sm_70)
+// CHK-PHASES-SM35-70: 21: sycl-post-link, {20}, ir, (device-sycl, sm_70)
+// CHK-PHASES-SM35-70: 22: backend, {21}, assembler, (device-sycl, sm_70)
+// CHK-PHASES-SM35-70: 23: assembler, {22}, object, (device-sycl, sm_70)
+// CHK-PHASES-SM35-70: 24: offload, "device-sycl (nvptx64-nvidia-cuda-sycldevice:sm_70)" {23}, object
+// CHK-PHASES-SM35-70: 25: offload, "device-sycl (nvptx64-nvidia-cuda-sycldevice:sm_70)" {22}, assembler
+// CHK-PHASES-SM35-70: 26: linker, {17, 18, 24, 25}, cuda-fatbin, (device-sycl)
+// CHK-PHASES-SM35-70: 27: clang-offload-wrapper, {26}, object, (device-sycl)
+// CHK-PHASES-SM35-70: 28: offload, "host-sycl (x86_64-unknown-linux-gnu)" {9}, "device-sycl (nvptx64-nvidia-cuda-sycldevice)" {27}, image
+
+/// Checks when requesting 2 CUDA gpu targets + SPIR
+// RUN: %clangxx -ccc-print-phases -std=c++11 -target x86_64-unknown-linux-gnu -fsycl \
+// RUN: -fsycl-targets=nvptx64-nvidia-cuda-sycldevice,spir64-unknown-unknown-sycldevice \
+// RUN: -Xsycl-target-backend "--cuda-gpu-arch=sm_35" -Xsycl-target-backend "--cuda-gpu-arch=sm_70" %s 2>&1 \
```

Comment:
I don't think this will work.  `-Xsycl-target-backend` will require the target triple when more than on `-fsycl-targets` is provided.